lib/display.ts

```diff
@@ -0,0 +1,289 @@
+    if (suggestions) {
+      result.push(suggestions);
+      result.push(includeSectionSeparator());
+    }
+
+    const userCTA = formatUserCTA(options);
+    if (userCTA) {
+      result.push(userCTA);
+    }
+
+    index += 1;
+  }
+
+  return result.join(BREAK_LINE);
+}
+
+function includeSectionSeparator(): string {
+  return BREAK_LINE;
+}
+
+function formatIssue(testResult: TestResult, issue: Issue): string {
+  const result: string[] = [];
+  const issueData = testResult.issuesData[issue.issueId];
+  const severity = capitalize(issueData.severity);
+  const pkg = issue.pkgName;
+  const color = getColor(issueData.severity);
+
+  const header = color(`✗ ${severity} severity vulnerability found in ${pkg}`);
+  const description = `  Description: ${issueData.title}`;
+  const info = `  Info: https://snyk.io/vuln/${issue.issueId}`;
+  const introduced = `  Introduced through: ${formatIntroduced(
+    issueData.from,
+  )}`;
+  const from = formatFrom(issueData.from);
+  const fixedIn = formatFixedIn(issue);
+
+  result.push(header);
+  result.push(description);
+  result.push(info);
+  result.push(introduced);
+  result.push(from);
+  if (fixedIn) {
+    result.push(fixedIn);
+  }
+  result.push("");
+
+  return result.join(BREAK_LINE);
+}
+
+function capitalize(word: string): string {
+  return word[0].toUpperCase() + word.slice(1);
+}
+
+function formatIntroduced(fromList: string[][]) {
+  const result: string[] = [];
+
+  for (const from of fromList) {
+    result.push(from[0]);
+  }
+
+  return result.join(", ");
+}
+function formatFrom(fromList: string[][]): string {
+  const result: string[] = [];
+  let counter = 0;
+  const max = 3;
+  for (const localFrom of fromList) {
+    if (counter >= max) {
+      break;
+    }
+    counter += 1;
+
+    result.push(`  From: ${localFrom.join(" > ")}`);
+  }
+  if (fromList.length > max) {
+    result.push(`  and ${(fromList.length = max)} more...`);
+  }
+
+  return result.join(BREAK_LINE);
+}
+
+function formatFixedIn(issue: Issue): string | undefined {
+  if (!issue.fixInfo || !issue.fixInfo.nearestFixedInVersion) {
+    return undefined;
+  }
+
+  return chalk.bold.green(`  Fixed in: ${issue.fixInfo.nearestFixedInVersion}`);
+}
+
+function formatMetadataSection(
+  scanResult: ScanResult,
+  testResult: TestResult,
+): string {
+  const result: string[] = [];
+  result.push(formatMetadataLine("Organization:", testResult.org));
+
+  const packageManager = scanResult.identity.type;
+  result.push(formatMetadataLine("Package manager:", packageManager));
+
+  const target: ContainerTarget = scanResult.target as ContainerTarget;
+  const projectName = target.image;
+  const image = target.image.replace("docker-image|", "");
+  result.push(formatMetadataLine("Project name:", projectName));
+  result.push(formatMetadataLine("Docker image:", image));
+
+  return result.join(BREAK_LINE);
+}
+
+function formatMetadataLine(header: string, info: string = ""): string {
+  const padLength = 19;
```

Comment:
I understand this comes from the CLI but what is the reason behind it being 19?

Reply:
I don't know. I'll try to give a better name, something more meaningful. But why it is 19 I don't know... :/